src/Mvc/Mvc.Abstractions/ref/Microsoft.AspNetCore.Mvc.Abstractions.netcoreapp.cs

```diff
@@ -459,6 +460,7 @@ public partial class BindingInfo
     {
         public BindingInfo() { }
         public BindingInfo(Microsoft.AspNetCore.Mvc.ModelBinding.BindingInfo other) { }
+        public bool? AllowEmptyInputInBodyModelBinding { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute] set { } }
```

Comment:
Given this is in BindingInfo, why not just call this AllowEmptyBody?

Reply:
BindingInfo tends to keep the same property names as the metadata that it copies from since it's effectively glue code. I'm not opposed to a shorter name, but it doesn't really benefit anyone given how we don't really expect anybody but Mvc to ever construct to read this.